msal/src/Microsoft.Identity.Client/Internal/PlatformPlugin.cs

```diff
@@ -50,8 +50,8 @@ private static void InitializeWebFactoryAndPlatform()
 
```

Comment:
Since WebUIFactory is internal, is this really needed?

Reply:
i reverted back and used your idea for an error message for a platformNotSupported exception.